<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointWin</class>
 <widget class="QWidget" name="JointWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
   <item>
    <widget class="JointImageWin" name="m_pJointImageWin" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>3</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="m_pBtnOpen1">
        <property name="text">
         <string>打开图1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pBtnOpen2">
        <property name="text">
         <string>打开图2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pBtnSwap">
        <property name="text">
         <string>交换</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>方向</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_pType"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="m_pBtnSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JointImageWin</class>
   <extends>QWidget</extends>
   <header location="global">jointimagewin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
